frame/evm/src/precompiles.rs

```diff
@@ -17,11 +17,24 @@
 
 //! Builtin precompiles.
 
-use sp_std::{cmp::min, vec::Vec};
+use sp_std::{cmp::{min, max}, vec::Vec};
 use sp_core::H160;
 use evm::{ExitError, ExitSucceed};
 use ripemd160::Digest;
 use impl_trait_for_tuples::impl_for_tuples;
+use sp_application_crypto::TryFrom;
+
+#[cfg(feature="ed25519")]
```

Comment:
```suggestion
#[cfg(feature = "ed25519")]
```

Minor styling nits. The rest of the codebase has this spacing.